db/seeds.rb

```diff
@@ -5,3 +5,8 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+User.delete_all
+
+frosty = User.create!(name: 'Frosty', email: 'snow@brr.com', birthdate: DateTime.new(2001,2,3))
```

Comment:
Do we want to set up Faker for these? Or not since we probably don't need a ton of users.

I can also help with mocks/stubs for testing

Reply:
I think we can use faker later on if we need more data or we may use it to create random eras/events